src/scratch_comment_bubble.js

```diff
@@ -0,0 +1,140 @@
+import * as Blockly from 'blockly/core';
+
+export class ScratchCommentBubble extends Blockly.comments.CommentView {
+  constructor(sourceBlock, anchor) {
+    super(sourceBlock.workspace);
+    this.sourceBlock = sourceBlock;
+    this.moveTo(anchor.x + 40, anchor.y - 16);
+    this.anchor = anchor;
+    this.disposing = false;
+    this.id = Blockly.utils.idGenerator.genUid();
+
+    const location = this.getRelativeToSurfaceXY();
+    this.anchorChain = Blockly.utils.dom.createSvgElement(
+        Blockly.utils.Svg.LINE,
+        {
+          x1: anchor.x,
+          y1: anchor.y,
+          x2: this.getSize().width / 2,
+          y2: 16,
+          style: `stroke: ${sourceBlock.getColourTertiary()}; stroke-width: 1`
+        },
+        this.getSvgRoot());
+    this.getSvgRoot().insertBefore(this.anchorChain, this.getSvgRoot().firstChild);
+    this.getSvgRoot().setAttribute(
+        'style', `--colour-commentBorder: ${sourceBlock.getColourTertiary()};`);
+
+    Blockly.browserEvents.conditionalBind(
+      this.getSvgRoot(),
+      'pointerdown',
+      this,
+      this.startGesture,
+    );
+    // Don't zoom with mousewheel; let it scroll instead.
+    Blockly.browserEvents.conditionalBind(
+      this.getSvgRoot(),
+      'wheel',
+      this,
+      (e) => {
+        e.stopPropagation();
+      },
+    );
+  }
+
+  setDeleteStyle(enable) {}
+  showContextMenu() {}
+  setDragging(start) {}
+  select() {}
+  unselect() {}
+
+  isMovable() {
+    return true;
+  }
+
+  moveDuringDrag(newLocation) {
+    this.moveTo(newLocation);
+  }
+
+  moveTo(xOrCoordinate, y) {
```

Comment:
Ah darn, I didn't realize there was a signature conflict between different versions of `moveTo` :/ That's my bad.

Reply:
No worries! Might be nice to clean up some time in the future, but not sure it's worth a breaking change.